### DISCUSSION TITLE
Absolute Beginner

### DISCUSSION THREAD

**Discussion by PeterLinu** [cboxdoerfer/fsearch#413] [Q&A]
Hiyall,
I just installed fsearch and don't know how to get started. Nothing intuitive works.
I have a NAS drive containing 3 HDDs containing music. I want to be able to search by song title.
Please help me get started.

**Comment by bernd-wechner** [comment]
Just a word of caution. If someone is working out of the goodness of their heart to develop something and share it, comments like "Nothing intuitive works." are not likely to go down well. You may weather it, they may bite their tongue and answer you politely and helpfully, but rest assured, it's a little abrasive. I haven't lift a finger to contribute to fsearch myself (yet) but I don't remember ever having any experience with it that wasn't easy and pleasing is all I can say.

So I'd suggest when you file and issue, as an absolute beginner, to also focus on the precise experience, and steps to reproduce your experience and not subjective judgements. Screenshots can be useful and more. But even a really well-meaning developer won't know where to start on an issue filed like this, bar maybe holding your hand and asking "did you try ..."    On Friday, September 16, 2022 at 02:24:39 PM GMT+10, PeterLinu ***@***.***> wrote:  
 
 


Hiyall,
I just installed fsearch and don't know how to get started. Nothing intuitive works.
I have a NAS drive containing 3 HDDs containing music. I want to be able to search by song title.
Please help me get started.

**Comment by cboxdoerfer** [comment]
> I want to be able to search by song title.

FSearch can't search for music tags. So unless the title is part of the file name, you can't search by song title.

**Reply by MajikSunshine** [reply]
This is a bad look for a program who's help menu sends the user to a developer forum, only to be scolded for asking a simple question. Some of that "heart" should be applied to helping people get started, not scaring them off. The title "Absolute Beginner" seems like a perfectly good post to look at if you are, like myself, just trying to get things going.
If this were my project I would delete this post entirely.

**Reply by cboxdoerfer** [reply]
Why exactly should I delete my post? It simply answers the question. There's no scolding at all.
